.circleci/config.yml

```diff
@@ -917,11 +917,11 @@ workflows:
   nightly-basic-suiterunner-regression:
     triggers:
       - schedule:
-          cron: "15 17,4 * * *"
+          cron: "45 1 * * *"
```

Comment:
Just to be sure, you are not going to merge with the schedule change for this and following, right?

Reply:
Absolutely not, please check my special note in the PR's description.